Jenkinsfile

```diff
@@ -200,13 +214,13 @@ pipeline {
                     }
                     post { always { retry(3) { deleteDir() } } }
                 }
-                stage('Unit with MPI') {
-                    agent any
+                stage('Linux Unit with Threading') {
```

Comment:
Shouldn't we also be running the header tests on the other platforms? (To be consistent with what you've said)

Reply:
For the record, we're pushing this off on to another issue / PR.